doc/moduleRelays.md

```diff
@@ -0,0 +1,78 @@
+# Module Relays
+
+This module will interact with the cRIO outputs, driving 24 Volts relays.
+It has a control named `controlRelays` and an indicator named `debugControlRelays` as an interface.
+
+## Control Relays
+
+The `controlRelays` is an 8-bit unsigned (U8) interger control.
+It is declared as a typedef enum like this:
+
+```
+{
+   NiFpga_mainFPGA_ControlU8_controlRelays = 0x18002,
+} NiFpga_mainFPGA_ControlU8;
+```
+Do not care about the 0x18002 value.
+It is created after the FPGA compilation.
+
+Here is an example how to write a value in this control using C/C++:
+
+```c 
+NiFpga_WriteU8(session, NiFpga_mainFPGA_ControlU8_controlRelays, 0);
+```
+
+where `session` handles a currently FPGA session, opened with:
+
+```c
+NiFpga_Session session;
+```
+and `0` is the `uint8_t` value wrote to the control.
+
+Session must be closed when no longer needed.
+
+## Control Bit Description
+
+The following table describes the meaning of each bit in the `controlRelay` control.
+
+| Bit |  0  |  1   |
+|:---:|:-----:|:-----:|
+|  0  | ILC Motor Power is Off | ILC Motor Power is On |
+|  1  | ILC Communication Power is Off  | ILC Communication is On |
+|  2  | Disable cRIO Interlock | Enable cRIO Interlock |
+|  3  | Default value of Reset Power Motor Breakers | Reset Power Motor Breakers |
+|  4  | Default value of Reset Communication Power Breakers | Reset Communication Power Breakers |
```

Comment:
After I put the "reset bit 3 and 4" to be 1, do I need to put them back to 0 in the next functional call? Describe this if you know what will happen. If not, add a note here to say the details will be clarified in a latter time.